Beta_Docs_20240808 (#986)
microsoftgraph/entra-powershell: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Confirm-EntraBetaDomain.md

# Confirm-EntraBetaDomain

## Synopsis

Validate the ownership of a domain.

## Syntax

```powershell
Confirm-EntraBetaDomain 
 -Name <String> 
 [-CrossCloudVerificationCode <CrossCloudVerificationCodeBody>] 
 [<CommonParameters>]
```

## Description

The `Confirm-EntraBetaDomain` cmdlet validates the ownership of an Microsoft Entra ID domain.

The work or school account needs to belong to at least the **Domain Name Administrator** Microsoft Entra role.

## Examples

### Example 1: Confirm the domain

```powershell
Connect-Entra -Scopes 'Domain.ReadWrite.All'
Confirm-EntraBetaDomain -Name Contoso.com
```

> This command will confirm your domain; changing the status to "Verified".

Reviewer: ```suggestion
This example verifies a domain and updates its status to `verified`.
```

### Example 2: Confirm the domain with a cross cloud verification code

```powershell
Connect-Entra -Scopes 'Domain.ReadWrite.All'
Confirm-EntraBetaDomain -Name Contoso.com -CrossCloudVerificationCode ms84324896
```

> This command will confirm your domain for dual federation scenarios.

Reviewer: ```suggestion
This example confirms a domain in dual federation scenarios.
```
